src/common.rs

```diff
@@ -835,11 +835,11 @@ pub struct DiskUsage {
 /// Enum describing the different status of a process.
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
 pub enum ProcessStatus {
-    /// ## Linux/FreeBSD
+    /// ## Linux
     ///
-    /// Waiting in uninterruptible disk sleep.
+    /// Interruptible task.
```

Comment:
The `ps` manpage uses the terminology "Idle kernel thread", perhaps we should use that?

Reply:
Let's go with it then.